interfaces/apparmor/export_test.go

```diff
@@ -108,3 +110,24 @@ func MockParserFeatures(f func() ([]string, error)) (resture func()) {
 		parserFeatures = old
 	}
 }
+
+// WritableMimicProfile writes apparmor rules for a writable mimic at the given path.
+func WritableMimicProfile(buf *bytes.Buffer, path string, assumedPrefixDepth int) {
+	GenWritableMimicProfile(func(f string, args ...interface{}) {
+		fmt.Fprintf(buf, f, args...)
+	}, path, assumedPrefixDepth)
+}
+
+// WritableFileProfile writes a profile for snap-update-ns for making given file writable.
+func WritableFileProfile(buf *bytes.Buffer, path string, assumedPrefixDepth int) {
+	GenWritableFileProfile(func(f string, args ...interface{}) {
+		fmt.Fprintf(buf, f, args...)
+	}, path, assumedPrefixDepth)
+}
+
+// WritableProfile writes a profile for snap-update-ns for making given directory writable.
+func WritableProfile(buf *bytes.Buffer, path string, assumedPrefixDepth int) {
+	GenWritableProfile(func(f string, args ...interface{}) {
+		fmt.Fprintf(buf, f, args...)
+	}, path, assumedPrefixDepth)
+}
```

Comment:
These three helpers seem unused (replaced by GenWritable* helpers directly)?

Reply:
Those helpers are actually tested :) All the test code calls `WriteProfile` and look at the resulting buffer. I didn't want to change that as those are super tedious to touch.

Reply:
Ooops, they are not tested directly. I removed them now.